tests/formats/test_bmsbk.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+from tests.test_lib import parse_build_compare_editor
+
+from mercury_engine_data_structures import samus_returns_data
+from mercury_engine_data_structures.formats.bmsbk import Bmsbk
+
+all_sr_bmsbk = [name for name in samus_returns_data.all_name_to_asset_id().keys()
```

Comment:
Looking at the failing tests there are assets which did not make it into the final game.